roles/mysql/tasks/main.yml

```diff
@@ -0,0 +1,199 @@
+---
+# tasks file for mysql
+#
+- name: MySQL | configure apt repositories
+  ansible.builtin.apt_key:
+    keyserver: keyserver.ubuntu.com
+    id: "0xF1656F24C74CD1D8"
+
+- name: MySQL | add apt repositories
+  ansible.builtin.apt_repository:
+    repo: deb [arch=amd64] https://mirrors.gigenet.com/mariadb/repo/10.2/ubuntu {{ ansible_distribution_release }} main
```

Comment:
This mirror keep changing.  I wonder if we should set this as a variable instead?